src/sections/ServiceMesh/home/service-mesh-focussed.js

```diff
@@ -0,0 +1,121 @@
+import styled from "styled-components";
+
+const MeshFocusWrapper = styled.div`
+   .main-cont {
+    display:flex;
+    flex-direction:column;
+    justify-content:center;
+    align-items:center;
+   } 
+    
+    .head-cont-2{
+        display:flex;
+        flex-direction:column;
+        justify-content:center;
+        align-items:center;
+        padding:20px 10px;
+    }
+    .head-cont-2 h1{
+        display:flex;
+        flex-direction:column;
```

Comment:
Same as above.
Instead os using `span` within `h1` use 2 `h1` tags. It should work.